src/queries/test-id.js

```diff
@@ -2,8 +2,9 @@ import {queryAllByAttribute, getConfig, buildQueries} from './all-utils'
 
 const getTestIdAttribute = () => getConfig().testIdAttribute
 
-const queryAllByTestId = (...args) =>
-  queryAllByAttribute(getTestIdAttribute(), ...args)
+function queryAllByTestId(...args) {
+  return queryAllByAttribute(getTestIdAttribute(), ...args)
+}
```

Comment:
ICYW: all of the other queries use named functions and the suggestion feature needs them to be named.